tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_repo_files.py

```diff
@@ -96,5 +97,7 @@ def test_setup_install_source_task_success(self):
 
     def test_setup_install_source_task_failure(self):
         with TemporaryDirectory() as temp_dir_name:
-            with self.assertRaises(SourceSetupError, msg="repo files not found"):
+            with pytest.raises(SourceSetupError) as cm:
                 SetUpRepoFilesSourceTask([temp_dir_name]).run()
+
+                assert str(cm.value) == "repo files not found"
```

Comment:
I don't think these checks are used, because you jump out of the blocks in the `run` method.

Reply:
Ahh, my mistake, thanks!

Reply:
Thanks for pointing that out. Fixed.